package-lock.json

```diff
@@ -16,7 +16,7 @@
         "gulp-concat": "^2.6.1"
       },
       "engines": {
-        "node": "> 7"
+        "node": ">=18"
```

Comment:
We're only changing the lockfile in this commit. Did you forget to add `package.json` to the commit here or did you manually edit `package-lock.json`?

In general the process should be:
- Edit `package.json`
- Run `npm install` to rebuild `package-lock.json`

Reply:
Fixed! https://github.com/DemocracyClub/electionleaflets/pull/255/commits/15bf573c409f9b6d71d98d7591a68eed585f5095